matrix/matrix_class.py

```diff
@@ -220,14 +227,15 @@ def inverse(self):
         determinant = self.determinant()
         return None if not determinant else self.adjugate() * (1 / determinant)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return str(self.rows)
 
-    def __str__(self):
+    def __str__(self) -> str:
         if self.num_rows == 0:
             return "[]"
         if self.num_rows == 1:
-            return "[[" + ". ".join(self.rows[0]) + "]]"
+            print(self.rows[0])

```

Comment:
I dont think using print is a good practise is algorithms.
If possible, please remove it.

Reply:
Removed it.